Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -867,14 +867,17 @@ extension RouteController: Router {
             case let .success(indexedResponse):
                 let response = indexedResponse.routeResponse
                 guard case let .route(routeOptions) = response.options else {
-                    //TODO: Can a match hit this codepoint?
-                    self.isRerouting = false; return
+                    self.isRerouting = false
+                    return announceReroutingError(with: ReroutingError.routeError)
```

Comment:
I find it a little bit confusing. It makes an impression like `announceReroutingError` does return something. 
```suggestion
                    announceReroutingError(with: ReroutingError.routeError)
                    return
```

Reply:
😅  This was a trick to save 1 line of code, but let's remove